config/kernel-objtool.m4

```diff
@@ -84,7 +93,32 @@ AC_DEFUN([ZFS_AC_KERNEL_OBJTOOL], [
 		],[
 			AC_MSG_RESULT(no)
 		])
+
+		AC_MSG_CHECKING([whether CONFIG_OBJTOOL_WERROR is defined])
+		ZFS_LINUX_TEST_RESULT([config_objtool_werror], [
+			AC_MSG_RESULT(yes)
+			AS_IF([test x$enable_objtool_werror = x],[
+				dnl # This could be made more adaptable.
+				AC_MSG_NOTICE([enable-objtool-werror undefined, disabling -Werror ])
+				OBJTOOL_DISABLE_WERROR=y
+				abs_objtool_binary=$kernelsrc/tools/objtool/objtool
+				dnl # Neither $abs_top_srcdir nor $top_srcdir are defined.
+				dnl # Hack around by using $PWD since we are in top src dir
+				dnl # while running configure.
+				abs_objtool_wrapper=$PWD/scripts/objtool-wrapper
+				AS_IF([test -x $abs_objtool_binary],[],[
```

Comment:
Assuming we can get `$top_srcdir` this would probably be more robust as a relative path.

Reply:
We are using it in `modules/Makefile`, will a relative path work there?

Reply:
On the other hand we could add the `--Werror` flag  if asked to.

Reply:
It would only make sense if we'd be objtool clean though. Maybe something to consider once that happens.